### PR TITLE
DG-9670: Upgrade vips/sharp, fix gif support

### PR DESCRIPTION
**Pull request by richardadjogah**

### CODE REVIEW

File: test/imageHelperTest.js [hunk 1]
```diff
@@ -39,12 +39,6 @@ describe('Image Helper Test', function() {
 		done();
 	});
 
-	it('correctly detects image extensions for conversion with fallback logic: gif -> jpeg', function(done) {
-		var imageExt = imageHelper.getImageExt('http://fake.website/test.gif', testBuffer);
-		assert.equal(imageExt, 'jpeg');
-		done();
-	});
```
Comment (erindethorne) [inline test/imageHelperTest.js]: Naive question - why is this test no longer relevant?

Comment (erindethorne) [inline test/imageHelperTest.js]: Is there a different unit test that would be appropriate instead? I just hate to see us bringing down test coverage if we don't have to.

Comment (richardadjogah) [inline test/imageHelperTest.js]: I wish, but at the moment we have no way of doing larger scope tests that actually check that images have been processed correctly.

**Comment by erindethorne** [comment]
:shipit:

**Pull request merged**